go.mod

```diff
@@ -1,10 +1,10 @@
-module github.com/lieut-data/mattermost-plugin-api
+module github.com/mattermost/mattermost-plugin-api
 
 go 1.13
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20191203094615-d0d6ce0a7014
-	github.com/pkg/errors v0.8.1
+	github.com/mattermost/mattermost-server/v5 v5.20.0
+	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.4.2
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
The version bumps are not stickily needed, but I did them anyway because it's not worth a separate PR